### ISSUE TITLE
Project Selection - Project File Name Must be Different

### ISSUE THREAD

**Issue by Urchmaney** [motion-canvas/motion-canvas#1076]
**Description**
While setting up multiple project in my repository, I was under the false assumption that if the path to the project is different it would work.
But found out after sometime that the file name is what's being used in motion canvas and not the path.
In my <b>vite.config.ts</b> file I had:
```
motionCanvas({ project: [
    "./src/blazer/project.ts",
    "./src/linked/project.ts"
  ]}
)
```
so I needed to change it to
```
motionCanvas({ project: [
    "./src/blazer/blazer_project.ts",
    "./src/linked/linked_project.ts"
  ]}
)
```
for it to work.

**How can it be improved?**
- Specify explicitly in the doc (under Getting Started/Configuration) that project file names must be different
- (OR) Allow the usage of path. If there is collision with the project file name, folder name can be used. (I can create a PR for this if it is considered)

**Comment by aarthificial** [comment]
Yes, using the entire file path as the url would be the desired solution as mentioned here: https://github.com/motion-canvas/motion-canvas/issues/297#issuecomment-1746172903
Feel free to open a PR if you're still interested

**Comment by aarthificial** [comment]
Closed by #1090
